src/core/Global.ttslua

```diff
@@ -857,6 +857,10 @@ function applyOptionPanelChange(id, state)
   -- option: Show chaos bag manager
   elseif id == "showChaosBagManager" then
     spawnOrRemoveHelper(state, "Chaos Bag Manager", {-67.8, 1.4, -49.5})
+
+  -- option: Show navigation overlay
+  elseif id == "showNavigationOverlay" then
+    spawnOrRemoveHelper(state, "jaqenZann's Navigation Overlay", {-11.7, 1.4, -15})
```

Comment:
We were going to remove the author names on helpers, right?

Reply:
Yep, that is the plan but not part of this PR.